.gitpod.dockerfile

```diff
@@ -0,0 +1 @@
+FROM gitpod/workspace-mysql
```

Comment:
@jywarren I think we can remove this file??

Reply:
possibly! want to try?

Reply:
@jywarren I checked gitpod after removing. Buil did getting failed so added it again.
Thanks!!